src/librustc/mir/interpret/allocation.rs

```diff
@@ -624,12 +622,14 @@ pub struct UndefMask {
 impl_stable_hash_for!(struct mir::interpret::UndefMask{blocks, len});
 
 impl UndefMask {
-    pub fn new(size: Size) -> Self {
+    pub const BLOCK_SIZE: u64 = 64;
+
+    pub fn new(size: Size, state: bool) -> Self {
```

Comment:
I infer that `state` here is interpreted as { `true` => defined, `false` => undefined }, (right?)

You might consider adding a comment above the header saying so. (My initial interpretation of "undef *mask*" was that if the bit is true, then it is undefined)